README.md

```diff
@@ -0,0 +1,54 @@
+# Fluentd Aggregator Docker Image
+
+[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/fluent/fluentd-aggregator?sort=semver)](https://hub.docker.com/r/fluent/fluentd-aggregator)
+![linux](https://img.shields.io/badge/os-linux-brightgreen)
+![amd64](https://img.shields.io/badge/arch-amd64-brightgreen)
+![arm64](https://img.shields.io/badge/arch-arm64-brightgreen)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+A [Fluentd](https://www.fluentd.org/) [OCI](https://opencontainers.org/) image to be used for log aggregation and based on the official [Fluentd Docker image](https://github.com/fluent/fluentd-docker-image) rebuilt as a multi-arch `linux/amd64` & `linux/arm64` image (until the official image support multi-arch).
+
+## Aggregation Changes
+
+To optimise _Fluentd_ for log aggregation the default `fluent.conf` file has been overwritten to allow logs to be forwarded and printed to stdout. Plugins have also been added to support the aggregation role.
+
+### Plugins
+
+The following plugins have been added to the base image.
+
+- [fluent-plugin-azure-loganalytics](https://github.com/yokawasa/fluent-plugin-azure-loganalytics)
+- [fluent-plugin-azurestorage-gen2](https://github.com/oleewere/fluent-plugin-azurestorage-gen2)
+- [fluent-plugin-cloudwatch-logs](https://github.com/fluent-plugins-nursery/fluent-plugin-cloudwatch-logs)
+- [fluent-plugin-concat](https://github.com/fluent-plugins-nursery/fluent-plugin-concat)
+- [fluent-plugin-datadog](https://github.com/DataDog/fluent-plugin-datadog)
+- [fluent-plugin-elasticsearch](https://docs.fluentd.org/output/elasticsearch)
+- [fluent-plugin-grafana-loki](https://github.com/grafana/loki/tree/main/clients/cmd/fluentd)
+- [fluent-plugin-opensearch](https://github.com/fluent/fluent-plugin-opensearch)
+- [fluent-plugin-prometheus](https://github.com/fluent/fluent-plugin-prometheus)
+- [fluent-plugin-record-modifier](https://github.com/repeatedly/fluent-plugin-record-modifier)
+- [fluent-plugin-rewrite-tag-filter](https://github.com/fluent/fluent-plugin-rewrite-tag-filter)
+- [fluent-plugin-route](https://github.com/tagomoris/fluent-plugin-route)
+- [fluent-plugin-s3](https://docs.fluentd.org/output/s3)
+- [fluent-plugin-sqs](https://github.com/ixixi/fluent-plugin-sqs)
+
+## Usage
+
+This image is available at [Docker Hub](https://hub.docker.com/r/fluent/fluentd-aggregator); the image version matches the _Fluentd_ version that it's based on.
```

Comment:
That last bit after the semi-colon is quite important: the versioning strategy is to follow fluentd version - what do you do if you need to do an update without changing the underlying fluentd version? Please capture the intent.

Reply:
See comment below.